firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -307,7 +310,11 @@ private final void doRegistrationInternal(boolean forceRefresh) {
     }
 
     triggerOnStateReached(prefs);
+    networkExecutor.execute(() -> run(forceRefresh));
+  }
 
+  private void run(boolean forceRefresh) {
```

Comment:
please give this a better name

Reply:
+1

How about this:
#sendRequestToFirebaseInstallationsServerApi
#syncWithFirebaseInstallationsServerApi 
#updateLocalAuthToken

Reply:
Updated this. PTAL.

Reply:
If possible I would recommend to always start a method name with a verb.

Also please remove the "fis" prefix:
a) "fis" is not a public term (so rather use "firebaseInstallations")
b) it's noise (we are in a class that is called FirebaseInstallations)

If you prefer your current name over the suggested ones, then I recommend to update it like this:
#sendNetworkCall
#makeNetworkCall